lib/rubocop/cop/rails/duplicate_association.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for associations that have been defined multiple times in the same file.
+      #
+      # @example
+      #
+      #   # bad
+      #   belongs_to :foo
+      #   belongs_to :bar
+      #   has_one :foo
+      #
+      #   # good
+      #   belongs_to :foo
+      #   has_one :bar
+      #
+      class DuplicateAssociation < Base
+        include RangeHelp
+        extend AutoCorrector
+
+        MSG = "Association `%<name>s` is defined multiple times. Don't repeat associations in a model."
```

Comment:
Can you tweak the message?

```suggestion
        MSG = "Association `%<name>s` is defined multiple times. Don't repeat associations."
```

Reply:
Done!